### PR TITLE
(Improvement) Improve how favorites' breaks lines

### PR DESCRIPTION
**Pull request by JasonMHasperhoven**
Before:
image

After:
image

### CODE REVIEW

File: src/components/FavoriteTradingPairs/index.js [hunk 1]
```diff
@@ -4,6 +4,19 @@ import FavoriteIcon from '../../ui/Icons/FavoriteIcon'
 
 import './style.css'
 
+const Pair = ({ pair }) => {
+  const [ccy, quote] = pair.split('/')
+
+  return [ccy, '/', quote].map(part => (
```
Comment (Blaumaus) [inline src/components/FavoriteTradingPairs/index.js:10]: I think it's better to use lodash `_map` here as Bhoomi proposed some time ago

File: src/components/FavoriteTradingPairs/index.js [hunk 2]
```diff
@@ -4,6 +4,19 @@ import FavoriteIcon from '../../ui/Icons/FavoriteIcon'
 
 import './style.css'
 
+const Pair = ({ pair }) => {
+  const [ccy, quote] = pair.split('/')
```
Comment (Blaumaus) [inline src/components/FavoriteTradingPairs/index.js:8]: Same as above: we should use lodash `_split`

**Review by Blaumaus — CHANGES_REQUESTED** [review]
Great job, with some minor proposals.

File: src/components/FavoriteTradingPairs/index.js (hunk 1 above)
Comment (JasonMHasperhoven) [reply src/components/FavoriteTradingPairs/index.js]: Done

**Review by Blaumaus — APPROVED** [review]

**Review by bhoomij — APPROVED** [review]

**Pull request merged**